Update README.md (#18)
canonical/content-cache-operator: content-cache-backends-config/README.md

Use links instead.
-->

# Content Cache Backends Config Operator

A [Juju](https://juju.is/) [subordinate](https://juju.is/docs/sdk/charm-taxonomy#heading--subordinate-charms) [charm](https://juju.is/docs/olm/charmed-operators) to the [Content Cache charm](https://charmhub.io/content-cache) which provides the Content Cache charm with the configuration required to expose a set of backend service behind caching capabilities of the Content Cache charm

> This charm should be integrated with the Content Cache charm to inject the charm configuration of this charm. For more information see the [Content Cache charm](https://charmhub.io/content-cache).

Reviewer: ```suggestion
This charm should be integrated with the Content Cache charm to inject the correct configurations into this charm. For more information see the [Content Cache charm](https://charmhub.io/content-cache).
```
(attempting to remove a "charm" from the sentence since there are so many instances in that particular sentence)

> ## Get Started

Reviewer: ```suggestion
## Get started
```
To adhere to the [style guide](https://docs.ubuntu.com/styleguide/en/#headings) recommendation to use sentence case in headings

> The Content Cache Backends Config Charm requires the deployment of the Content Cache charm. To begin, refer to the [Content Cache tutorial](https://github.com/canonical/content-cache-operator/blob/main/content-cache/docs/tutorial/quick-start.md) for step-by-step instructions.

Author: had to use github links instead of charmhub documentation links in a couple of places since the charmhub docs are not updated/available yet. @erinecon i hope that's okay

Reviewer: Thank you for checking -- I am okay with that!

### Basic Operations

Some of the Content Cache Backends Config Charm configurations include:

- backends
- backends-path
- fail-timeout
- hostname

and many more. For the complete list of configurations and their usage, refer to the [Charmhub Documentation](https://charmhub.io/content-cache-backends-config/configurations).

## Integrations

The Content Cache Backends Config Charm needs to be integrated with the Content Cache charm. For more details regarding the integration, refer to the [the Charmhub documentation](https://charmhub.io/content-cache-backends-config/integrations).

## Learn more

- [Read more](https://charmhub.io/content-cache-backends-config)
- [Troubleshooting](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)

> 

Author: since it does not have any workload charm, i wasn't sure what to put for developer docs

> 

Reviewer: Ah ok, thanks for explaining! No change needed then :)

## Project and community

The Content Cache Backends Config Operator is a member of the Ubuntu family. It is an
open source project that warmly welcomes community projects, contributions,
suggestions, fixes and constructive feedback.
* [Code of conduct](https://ubuntu.com/community/code-of-conduct)

> * [Get support](https://discourse.charmhub.io/)

Author: done

> * [Contribute](https://github.com/canonical/content-cache-operator/blob/main/content-cache-backends-config/CONTRIBUTING.md)

Author: ditto

Reviewer: Thank you for pointing this out. I'm fine with linking the `CONTRIBUTING.md` file here

Use links instead.
-->

# Content Cache Backends Config Operator

A [Juju](https://juju.is/) [subordinate](https://juju.is/docs/sdk/charm-taxonomy#heading--subordinate-charms) [charm](https://juju.is/docs/olm/charmed-operators) to the [Content Cache charm](https://charmhub.io/content-cache) which provides the Content Cache charm with the configuration required to expose a set of backend services behind caching capabilities of the Content Cache charm.

> This charm should be integrated with the Content Cache charm to inject the correct configurations into this charm. For more information see the [Content Cache charm](https://charmhub.io/content-cache).

Reviewer 2: ```suggestion
This charm should be integrated with the Content Cache charm to inject the correct configurations into the content cache charm. For more information see the [Content Cache charm](https://charmhub.io/content-cache).
```
